sources/views/contacts/formInContacts.js

```diff
@@ -1,21 +1,45 @@
 import {JetView} from "webix-jet";
+import { contacts } from "models/contacts.js";
 
 export default class FormInContactsView extends JetView {
 	config() {
+		const _ = this.app.getService("locale")._;
 		return {
 			view: "form",
+			localId: "myForm",
 			autoheight: false,
 			elements: [
-				{view: "text", label: "Name"},
-				{view: "text", label: "E-mail"},
+				{view: "text", label: _("Name"), name: "Name"},
+				{view: "text", label: _("E-mail"), name: "Email"},
+				{view: "combo", label: _("Countries"), name: "Country", value:"",
+					options:[{value:1}, {value:2}, {value:3}]},
```

Comment:
country and statuses combo boxes need to reflect statuses and countries data from models and have the correct templates for suggest body to show the values, not ids